### PR TITLE
Format

### PR DESCRIPTION
**Pull request by iliailmer**
resolving conflicts from #105

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
Looks like the formatter didn't run here 😅

**Comment by iliailmer** [comment]
@ChrisRackauckas there was also a "<<<<" left over, I think.

**Comment by iliailmer** [comment]
@ChrisRackauckas no wonder it did not run: i did not even add the yml file 😄

**Comment by pogudingleb** [comment]
@iliailmer  Just wanted to point out that not all formatting checks are passing.

**Comment by iliailmer** [comment]
> @iliailmer Just wanted to point out that not all formatting checks are passing.

Thank you! I just noticed and started fixing the remaining files.

File: benchmarking/benchmarks.jl [hunk 1]
```diff
@@ -1,260 +1,346 @@
...
-    )
+    Dict(:name => "Modified LV for testing",
+         :ode => @ODEmodel(x1'(t)=(a + b) * x1(t) - c * x1(t) * x2(t),
+                           x2'(t)=-a * b * x2(t) + d * x1(t) * x2(t),
+                           y1(t)=x1(t))),
+    Dict(:name => "SIWR original",
+         :ode => @ODEmodel(function S'(t)
+                               mu - bi * S(t) * I(t) - bw * S(t) * W(t) - mu * S(t) +
+                               a * R(t)
+                           end,
```
Comment (pogudingleb) [inline benchmarking/benchmarks.jl:10]: Why do we do this?

File: src/ODE.jl [hunk 2]
```diff
@@ -28,20 +26,28 @@ struct ODE{P}
...
+function add_outputs(ode::ODE{P}, extra_y::Dict{String, <:RingElem}) where {P <: MPolyElem}
+    new_var_names = vcat(collect(map(var_to_str, gens(ode.poly_ring))),
+                         collect(keys(extra_y)))
     new_ring, new_vars = Nemo.PolynomialRing(base_ring(ode.poly_ring), new_var_names)
 
-    new_x_eqs = Dict{P, Union{P, Generic.Frac{P}}}(parent_ring_change(x, new_ring) => parent_ring_change(f, new_ring) for (x, f) in ode.x_equations)
-    new_y_eqs = Dict{P, Union{P, Generic.Frac{P}}}(parent_ring_change(y, new_ring) => parent_ring_change(g, new_ring) for (y, g) in ode.y_equations)
-    extra_y_eqs = Dict{P, Union{P, Generic.Frac{P}}}(str_to_var(y, new_ring) => parent_ring_change(g, new_ring) for (y, g) in extra_y)
+    new_x_eqs = Dict{P, Union{P, Generic.Frac{P}}}(parent_ring_change(x, new_ring) => parent_ring_change(f,
+                                                                                                         new_ring)
```
Comment (pogudingleb) [inline src/ODE.jl:43]: This is a very tricky indentation, can we make it other way?

File: src/ODE.jl [hunk 3]
```diff
@@ -59,17 +65,23 @@ Input:
...
+                     for v in new_vars)
     merge!(eval_dict, Dict(p => small_ring(val) for (p, val) in param_values))
 
-    return ODE{P}(
-        Dict{P, Union{P, Generic.Frac{P}}}(eval_at_dict(v, eval_dict) => eval_at_dict(f, eval_dict) for (v, f) in ode.x_equations),
-        Dict{P, Union{P, Generic.Frac{P}}}(eval_at_dict(v, eval_dict) => eval_at_dict(f, eval_dict) for (v, f) in ode.y_equations),
-        [eval_at_dict(u, eval_dict) for u in ode.u_vars]
-    )
+    return ODE{P}(Dict{P, Union{P, Generic.Frac{P}}}(eval_at_dict(v, eval_dict) => eval_at_dict(f,
+                                                                                                eval_dict)
```
Comment (pogudingleb) [inline src/ODE.jl:79]: Same here

File: src/ODE.jl [hunk 4]
```diff
@@ -295,13 +301,21 @@ macro ODEmodel(ex::Expr...)
...
     y_vars = Set()
     x_dict_create_expr = :($x_dict = Dict{
-        StructuralIdentifiability.Nemo.fmpq_mpoly,
-        Union{StructuralIdentifiability.Nemo.fmpq_mpoly, StructuralIdentifiability.AbstractAlgebra.Generic.Frac{StructuralIdentifiability.Nemo.fmpq_mpoly}}
-    }())
+                                          StructuralIdentifiability.Nemo.fmpq_mpoly,
+                                          Union{StructuralIdentifiability.Nemo.fmpq_mpoly,
+                                                StructuralIdentifiability.AbstractAlgebra.Generic.Frac{
+                                                                                                       StructuralIdentifiability.Nemo.fmpq_mpoly
+                                                                                                       }
```
Comment (pogudingleb) [inline src/ODE.jl:308]: Can we have the bracket not as far?

File: src/global_identifiability.jl [hunk 5]
```diff
@@ -289,23 +287,24 @@ end
 
 For the io_equation and the list of all parameter variables, returns a set of *raw* *generators of a field of all functions of parameters
 """
-function extract_identifiable_functions_raw(
-    io_equations::Array{P,1},
-    parameters::Array{P,1},
-) where P <: MPolyElem{fmpq}
+function extract_identifiable_functions_raw(io_equations::Array{P, 1},
+                                            parameters::Array{P, 1}) where {
+                                                                            P <:
```
Comment (pogudingleb) [inline src/global_identifiability.jl:292]: Can we have it not that far?

File: benchmarking/benchmarks.jl (hunk 1 above)
Comment (iliailmer) [reply benchmarking/benchmarks.jl:10]: oh, that was an accidental; formatter thinks this is a function definition. I will fix that

**Comment by pogudingleb** [comment]
#152 has been merged  addressing `@ODEmodel` issues, so I believe that this closes the formatting issue.

**Pull request closed without merging**